Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 #!groovy
 
 // koji-build: trigger if the dist tag matches this regex
-def triggerDistTagRegEx = '^f[3-9]{1}[0-9]{1}$'
+def triggerDistTagRegEx = '^(f[3-9]{1}[0-9]{1}|epel9|epel10.0)$'
```

Comment:
Why is the dist-tag (in messages) for EPEL so weird? Isn't dist-tag just `el9` or `el10` (?) This looks like a bug...

In any case, expecting `epel10.0` is probably not a good idea (imo). What happens when RHEL 10.1 comes around? Will the dist-tag change to `epel10.1`, and the trigger will stop working?

Reply:
Thanks @carlwgeorge, please see now.